ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -74,7 +74,7 @@ class TooltipView: UIView {
 
     private let messageLabel: UILabel = {
         let label = Label(style: Constants.messageLabelTextStyle)
-        label.textColor = Colors.Tooltip.text
+        label.textColor = UIColor(dynamicColor: label.fluentTheme.aliasTokens.colors[.foregroundInverted1])
```

Comment:
isn't this kind of duplicating updateMessageLabelColor ?

Reply:
Kind of. I can change `updateMessageLabelColor` and use it here as well.

Reply:
why is it needed? by the time label is shown in the window technically it should be already called? have you tried debugging line by line to see the view life cycle?

Reply:
Ah I think I see what you mean. `updateMessageLabelColor` is only getting called when there is a theme change.

Reply:
instead of here then... should be it below to line 141 similar to how you handle background color update of the tooltip?

Reply:
Sounds good, I'll do that!

Reply:
yeah it seems like all the color updating logic can be group to one function that is used by init and themechange notification.